### PR TITLE
Reviewed tutorial steadyCom

### PR DESCRIPTION
**Pull request by almut-heinken**
Reviewed tutorial steadyCom, only minor edits. Note however, that I got infeasibility issues sometimes, but this was not consistent.

**I hereby confirm that I have:**

- [X] Tested my code on my own machine
- [X] Followed the guidelines in the [Contributing Guide](https://opencobra.github.io/cobratoolbox/docs/contributing.html)
- [X] Selected `develop` as a target branch (top left drop-down menu)

*(Note: You may replace [] with [X] to check the box)*

### CODE REVIEW

**Comment by almut-heinken** [comment]
Please reject the mat.files, they were pushed by accident!

File: tutorials/steadyCom/tutorial_steadyCom.m [hunk 1]
```diff
@@ -52,16 +52,20 @@
...
-%% 
-% 
 %% PROCEDURE
 %% Model Construction
-% Load the _E. coli_ iAF1260 model in the COBRA toolbox:
+% Load the _E. coli_ iAF1260 model in the COBRA toolbox. Please set the path 
+% to the model:
 
-load('iAF1260.mat', 'iAF1260');
+load('YOUR_PATH_TO_COBRA/cobratoolbox/test/models/iAF1260.mat', 'iAF1260');
```
Comment (laurentheirendt) [inline tutorials/steadyCom/tutorial_steadyCom.m]: May you put:
```Matlab
global CBTDIR
model = readCbModel([CBTDIR filesep 'test' filesep 'models' filesep 'iAF1260.mat']);
```

**Comment by almut-heinken** [comment]
How do I remove the POA folder? iAF1260 is not a SBML file, it's a mat file. And there are two files with the name in the toolbox.

**Comment by laurentheirendt** [comment]
Despite `iAF1260` being a `.mat` file, it is recommended to use `readCbModel`, which checks and loads the model.

**Comment by laurentheirendt** [comment]
What is `tutorials/steadyCom/POA/EcCom_GR0.52_POApre.mat`?

I will add a line in `.gitignore` so that these files will be ignored in the future.

**Comment by almut-heinken** [comment]
I  now got rid of these files and edited the code as suggested.

**Pull request merged**